v2/pkg/store/store.go

```diff
@@ -87,10 +87,12 @@ func (m *MemoryStore) Delete(ctx context.Context, d digest.Digest) error {
 
 // Info returns the info for a specific digest
 func (m *MemoryStore) Info(ctx context.Context, d digest.Digest) (ccontent.Info, error) {
+	m.labels.l.RLock()
 	info := ccontent.Info{
 		Digest: d,
 		Labels: m.labels.labels[d],
 	}
+	m.labels.l.Unlock()
```

Comment:
this should be `RUnlock()`, right?

Reply:
Thanks! You're right, it's a typo. Fixed.